specification/overview.md

```diff
@@ -26,21 +26,58 @@ This specification is designed to be used by three major groups:
 
 The FOCUS working group will develop an open-source specification for billing data. The schema will define data dimensions, metrics, a set of attributes about billing data, and a common lexicon for describing billing data.
 
-## Design Notes
+## Design Principles
 
 The following principles were considered while building the specification.
 
-### Provider-Neutral Approach by Default
+### FOCUS is an iterative, living specification
 
-While the schema, naming, terminology and attributes of many providers were reviewed during development, this specification aims to be provider-neutral. In some cases, the approach may closely resemble one or more provider's implementation, while in other cases, the approach might be new. In all cases, the FOCUS group (community composed of FinOps practitioners, Cloud and SaaS providers and FinOps vendors) will attempt to prioritize alignment with the FinOps [Framework][FODOF] and [Capabilities][FODOFC].
+The goal is not to get to a complete, finished specification in one pass. Releasing incremental iterations of the specification will provide value to practitioners and allow feedback as the specification develops.
 
 ### Working Backwards
 
-The FOCUS group working on the specification aims to work backwards from essential FinOps capabilities that practitioners need to perform to prioritize the dimensions, metrics and the attributes of the billing data that should be defined in the specification to fulfill that capability. Some of the enabled capabilities will be documented in the [Appendix: FinOps Capabilities Enabled by FOCUS (TODO)](#finopscapabilitiesenabledbyfocus).
+* The FOCUS group working on the specification aims to work backwards from essential FinOps capabilities that practitioners need to perform to prioritize the dimensions, metrics and the attributes of the cost and usage data that should be defined in the specification to fulfill that capability.
+* Be FinOps scenario-driven. Define columns that answer scenario questions; don't look for scenarios to fit a column, each column must have a use case.
+* Don't add data just because it can be added.
+
+### Ease of adoption is important
+
+* When defining the specification, consideration should be made to existing data already in the major providers' data sources.
```

Comment:
Who are the "major providers"?  (I know the answer, of course... but the document doesn't appear to make any declaration about this.  It assumes the reader already knows.)  While I can respect the sensitive nature of the exercise, I think it would be worth defining this term.